docs/01-Introduction.md

```diff
@@ -1,5 +1,55 @@
-# 1.0 - Introduction
+# 1.0 - Introduction 
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. 
+These documents assume you have some
+knowledge regarding LORIS and a functioning installation. For information
+regarding LORIS itself, please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is responsible for the processing, visualizing, and archiving
+of uploaded MRI scans. It expects an uploaded, compressed file containing
+several [DICOM][2] files, processes this into [MINC][3] and [NII][4]
+files, and then archives these files onto the server. Knowledge of
+these file formats can be helpful, but are not necessary for using or
+installing LORIS-MRI.
+
+LORIS-MRI allows you to easily organize and archive your MRI data and links it with 
+ corresponding behavioral data in LORIS. Scans can be viewed and quality controlled through the user's web browser, facilitating collaboration between radiologists, clinicians and researchers. 
+
+### 1.2 How does LORIS-MRI work?
+![user_story](images/user_story.png)
+First, users upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging Uploader.
+LORIS requires that the uploaded file name follow the naming convention PSCID_CANDID_VISIT-LABEL.
+Assuming the upload went through successfully, an entry is added to the menu of Imaging Uploader
+and the upload's status throughout the pipeline can then be tracked by
+the user through the Log Viewer.
+
```

Comment:
Deidentification of imaging datasets; please add a section on it; the text can be copied from:
https://github.com/aces/Loris-MRI/pull/219/commits/eaf9979ed91d1d47d46e7764d60964a95c7102c8

Reply:
Here you only comment on the filename, but also the DICOM headres need to be de-identified (please check the commit in my earlier comment for what can be added).